src/azure-cli/azure/cli/command_modules/redis/_help.py

```diff
@@ -69,6 +74,9 @@
   - name: Deploying a Premium Azure Cache for Redis inside an existing Azure Virtual Network
     text: az redis create --location westus2 --name MyRedisCache --resource-group MyResourceGroup --sku Premium --vm-size p1 --subnet-id "/subscriptions/{subid}/resourceGroups/{resourceGroupName}/providers/Microsoft.{Network|ClassicNetwork}/virtualNetworks/vnet1/subnets/subnet1"
     crafted: true
+  - name: Deploying a Premium Azure Cache for Redis with Microsoft Entra Authentication configured
+    text: az redis create --location westus2 --name MyRedisCache --resource-group MyResourceGroup --sku Premium --vm-size p1 --redis-configuration @"config_enable-aad.json"
+    crafted: true
```

Comment:
You don't need to add `crafted: true` for manually written examples

Reply:
Removed crafted for the example